test/test_type_promotion.py

```diff
@@ -120,6 +124,87 @@ def test_promotion(dtype_float, dtype_double):
         other = torch.tensor(5.5, dtype=torch.double, device=device)
         self.assertEqual((a + other).dtype, torch.complex64)
 
+        def make_scalar(dtype):
+            return make_tensor((), dtype=dtype, device=device)
+
+        def make_1d_tensor(dtype):
+            return make_tensor((3,), dtype=dtype, device=device)
+
+        def complex_scalar_tensor_test(s, t, is_float_tensor):
```

Comment:
why is `is_float_tensor` an argument and not programmatically determined from `t.dtype`?

Reply:
Missed that. Thanks!